src/service/FeatureToggleService/FeatureToggleController.ts

```diff
@@ -0,0 +1,11 @@
+import { APIClient, APIMapping } from '../../http';
+
+export class FeatureToggleController extends APIClient {
+    constructor() {
+        super(APIMapping.featureToggleService);
+    }
+
+    async feature(featureName: string) {
+        return await this.invokeApiWithErrorHandling<boolean>(`/features/${featureName}`, 'GET');
+    }
```

Comment:
GET Requests die etwas zurück geben, haben wir immer mit `fetch` geprefixed. Ein JSDoc Kommentar wäre auch schön :) 

```suggestion
    async fetchFeatureState(featureName: string) {
        return await this.invokeApiWithErrorHandling<boolean>(`/features/${featureName}`, 'GET');
    }
```

Reply:
Danke für den Hinweis. Hab es umbenannt und Kommentar hinzugefügt.